pkgs/applications/window-managers/notion/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  enableXft ? true, fontconfig ? null,
+  patches ? [],
+  fallbackFont ? "Sans",
+  stdenv, fetchurl,
+  lua, gettext, groff,
+  pkgconfig, busybox,
+  x11, libXinerama, libXrandr, libX11
+}:
+
+assert enableXft -> fontconfig != null && fallbackFont != null;
+
+let
+  xftPatch = "./notion-xft_nixos.diff";
+  name     = "notion";
+  version  = "3-2015061300";
+in
+stdenv.mkDerivation {
+
+  inherit name;
+  inherit version;
+  meta = with stdenv.lib; {
+    description = "Tiling tabbed window manager, follow-on to the ion window manager";
+    homepage = http://notion.sourceforge.net;
+    platforms = platforms.linux;
+    license   = licenses.notion_lgpl;
+    maintainers = [maintainers.jfb];
+  };
+  src = fetchurl {
+    url = https://github.com/raboof/notion/archive/3-2015061300.tar.gz;
+    sha256 = "3c9d9f35a9fb0d17c263b76fe28f7a1a4a05b7d6140545524cc1effd98c5c305";
+  };
+
+  patches = patches ++ stdenv.lib.optional enableXft xftPatch;
```

Comment:
The patch file must NOT be a string, also just put it inline: `patches = patches ++ stdenv.lib.optional enableXft ./notion-xft_nixos.diff;`